### DISCUSSION TITLE
Is clearable icon working properly?

### DISCUSSION THREAD

**Discussion by fjalcarranza** [dotnetcore/BootstrapBlazor#2804] [Q&A] [answered]
Good afternoon,

I'm testing  select control with `IsClearable="true"` but it happens nothing.
Am I doing it right or it needs to attach to an event (I inspect the code and I didn't see any public event)?

My code is as simple as:

html side:
`<Select class="my-3" IsVirtualize="true" OnQueryAsync="OnQueryAsync" @bind-Value="prequelItem" ShowSearch="true" IsClearable="true"></Select>`

C# side:
```
private IEnumerable<SelectedItem> sequelItems = Enumerable.Empty<SelectedItem>();

private Task<QueryData<SelectedItem>> OnQueryAsync(VirtualizeQueryOption option)
{
    var items = animeList.Select(item => new SelectedItem()
        {
            Value = item.Id.ToString(),
            Text = item.Name
        });

    if (!string.IsNullOrWhiteSpace(option.SearchText))
    {
        items = animeList
        .Where(item => item.Name.Contains(option.SearchText, StringComparison.OrdinalIgnoreCase) || item.OriginalName.Contains(option.SearchText, StringComparison.OrdinalIgnoreCase) || item.OtherNames.Contains(option.SearchText, StringComparison.OrdinalIgnoreCase))
        .Select(item => new SelectedItem()
            {
                Value = item.Id.ToString(),
                Text = item.Name
            });
    }

    return Task.FromResult(new QueryData<SelectedItem>()
        {
            Items = items,
            TotalCount = items.Count()
        });
}
```

I also read the code on `https://www.blazor.zone/select` and it doesn't work either.

I kindly ask for any help.

Regards.

**Comment by ArgoZhang** [comment]
@fjalcarranza sorry for the delayed reply. i will check it and give feedback later

**Comment by ArgoZhang** [comment, marked as answer]
@fjalcarranza sure. this is a bug. I will fix it in the next release.

**Reply by fjalcarranza** [reply]
Thanks for your reply, I'm looking forward the next release.

Regards.
